vyper/parser/parser.py

```diff
@@ -163,12 +163,10 @@ def parse_other_functions(o,
 
 # Main python parse tree => LLL method
 def parse_tree_to_lll(
```

Comment:
Is this function appropriately named?

Reply:
I'd expect this function to take an ast node as an argument

Reply:
`GlobalContext` actually contains all of the information from the AST, so it isn't necessary to pass in the AST itself.